content/kubernetes/deployment/openshift/openshift-operatorhub.md

```diff
@@ -18,51 +16,37 @@ aliases: [
 
 ]
 ---
-The OperatorHub is a feature of OpenShift 4.x that enables browsing a catalog
-of open-source and vendor-provided operators.
-You can install operators from the operator catalog in the OpenShift administrative interface with a few simple steps.
 
-You can install the operator via the Command-Line Interface (CLI), but the
-OperatorHub is an easy way to help you to manage the operator and apply
-upgrades. It also provides an easy way to create clusters.
+Redis Enterprise for Kubernetes can be deployed and administered entirely from the OpenShift command-line interface (CLI). For those who prefer a user interface (UI), OpenShift also provides the OperatorHub, a web console interface where you can install operators and create custom resources.
 
-The operator installation process is:
+For details on the OperatorHub, see the [OpenShift documentation](https://docs.openshift.com/container-platform/4.11/operators/index.html).
 
-1. Install the operator from the OperatorHub.
-1. Use a Custom Resource Definition (CRD) to create clusters from the Operator view
-   in the OperatorHub.
-1. Create databases in Redis Enterprise.
+## Prepare the cluster
 
-Now the databases are exposed as Kubernetes services accessible by other
-application workloads in your K8s cluster.
+The Redis Enterprise pods must run in OpenShift with privileges set in a [Security Context Constraint](https://docs.openshift.com/container-platform/4.4/authentication/managing-security-context-constraints.html#security-context-constraints-about_configuring-internal-oauth). This grants the pod various rights, such as the ability to change system limits or run as a particular user.
 
-## Preparing the cluster
+{{<warning>}}
+The security context constraint for the operator
+([scc.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/openshift/scc.yaml)) must be installed before the operator can create any clusters.
```

Comment:
```suggestion
You must install the security context constraint for the operator
([scc.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/openshift/scc.yaml)) before it can create any clusters.
```